eras/shelley/impl/src/Cardano/Ledger/Shelley/Governance.hs

```diff
@@ -87,6 +89,18 @@ class
   -- | Lens for accessing the previous protocol parameters
   prevPParamsGovStateL :: Lens' (GovState era) (PParams era)
 
+  -- | Getter for accessing the future protocol parameters.
+  --
+  -- This getter is only reliable and efficient 2 stability before the end of the
+  -- epoch. Depending on the era, if called earlier in the epoch, it will either produce
+  -- unreliable results or getting those results will be somewhat costly.
+  --
+  -- Whenever called at the earliest two stability before the end of the epoch, then the
+  -- results will be 100% reliable and they will contain either a `Just` value with the
+  -- new `PParams`, when there was an update proposed and `Nothing` whenever PParams will
```

Comment:
"proposed" does not necessarily carry the connotation that it was accepted/reached quorum?

Reply:
It is only about the Haddocks, thanks for clarifying that "proposed" here means "accepted"/"reached the quorum". I think it indeed makes sense to include that more detailed explanation in the Haddocks, thanks :+1:

Reply:
Replaced the implementation and documentation. Should be much more clear now.